unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/PlacesSubSection/PlacesSubSectionMenu/PlacesAPIController.cs

```diff
@@ -17,6 +17,18 @@ public interface IPlacesAPIController
     /// </summary>
     /// <param name="OnCompleted">It will be triggered when the operation has finished successfully.</param>
     void GetAllFavorites(Action<List<HotSceneInfo>> OnCompleted);
+
+    /// <summary>
+    /// Add a place to favorites
+    /// </summary>
+    /// <param name="placeUUID">UUID to identify the place.</param>
+    void AddFavorite(string placeUUID);
+
+    /// <summary>
+    /// Remove a place from favorites
+    /// </summary>
+    /// <param name="placeUUID">UUID to identify the place.</param>
+    void RemoveFavorite(string placeUUID);
```

Comment:
I didn't see any client of this public functions, so please remove them (be following YAGNI principle and adding new things only when they are needed). Also, remember that [_"PSA: Not every single type in your C# codebase needs to implement an interface. This is by far the most abused practise by C# developers."_](https://www.linkedin.com/posts/nick-chapsas_dotnet-dotnetdeveloper-csharp-activity-7026207482607796224-GREi?utm_source=share&utm_medium=member_desktop) Nick Chapsas. We agreed to only add them (and respective public methods inside it) when there are some specific clients (or for mocking in tests)